SchematicNeo4j/SchematicNeo4j/NodeKey.cs

```diff
@@ -94,17 +107,19 @@ public static class NodeKey
         /// <param name="type"></param>
         /// <param name="session"></param>
         /// <returns></returns>
-        public static bool Exists(Type type, ISession session) {
-            return session.ReadTransaction(tx => Exists(type, tx));
+        public static bool Exists(Type type, ISession session)
+        {
+            return session.ExecuteWrite(tx => Exists(type, tx));

```

Comment:
Should this be an `ExecuteRead` ?

Reply:
Yes it should, will be added in next PR.